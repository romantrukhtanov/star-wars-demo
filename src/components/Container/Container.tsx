import type { ReactNode } from 'react';

import styles from './Container.module.css';

type Props = {
  children: ReactNode;
};

export function Container({ children }: Props) {
  return (
    <div className={styles.container}>
      {children}
    </div>
  );
}
