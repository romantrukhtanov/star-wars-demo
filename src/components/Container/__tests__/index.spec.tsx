import { describe, expect, it } from 'vitest';

import { render } from '@/tests/test-utils';

import { Container } from '../Container';

describe('Container', () => {
  it('render component', () => {
    const { baseElement } = render(<Container>Some content</Container>);
    expect(baseElement).toMatchSnapshot();
  });
});
