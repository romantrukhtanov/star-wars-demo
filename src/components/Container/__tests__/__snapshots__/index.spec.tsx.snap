// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Container > render component 1`] = `
<body>
  <div>
    <div
      class="_container_523845"
    >
      Some content
    </div>
  </div>
</body>
`;
